src/KeyedFrames.jl

```diff
@@ -163,23 +169,25 @@ function Base.append!(kf::KeyedFrame, data)
     return kf
 end
 
-function deleterows!(kf::KeyedFrame, ind)
+function DataFrames.deleterows!(kf::KeyedFrame, ind)
     deleterows!(frame(kf), ind)
     return kf
 end
 
-deletecols!(kf::KeyedFrame, ind::Union{Integer, Symbol}) = deletecols!(kf, [ind])
-deletecols!(kf::KeyedFrame, ind::Vector{<:Integer}) = deletecols!(kf, names(kf)[ind])
+@deprecate deletecols!(kf::KeyedFrame, ind) select!(kf, ind)
+
+DataFrames.select!(kf::KeyedFrame, ind::Union{Integer, Symbol}) = select!(kf, [ind])
+DataFrames.select!(kf::KeyedFrame, ind::Vector{<:Integer}) = select!(kf, names(kf)[ind])
 
-function deletecols!(kf::KeyedFrame, ind::Vector{<:Symbol})
-    deletecols!(frame(kf), ind)
+function DataFrames.select!(kf::KeyedFrame, ind::Vector{<:Symbol})
+    select!(frame(kf), Not(ind))
```

Comment:
why is there a `Not` here?

Reply:
I think this is an error 
```julia
julia> df = DataFrame(a=1:3, b='a':'c');

julia> kf = KeyedFrame(df, names(df));

julia> select!(kf, :a)
3×1 KeyedFrame
│ Row │ b    │
│     │ Char │
├─────┼──────┤
│ 1   │ 'a'  │
│ 2   │ 'b'  │
│ 3   │ 'c'  │

julia> df = DataFrame(a=1:3, b='a':'c');

julia> select!(df, :a)
3×1 DataFrame
│ Row │ a     │
│     │ Int64 │
├─────┼───────┤
│ 1   │ 1     │
│ 2   │ 2     │
│ 3   │ 3     │
```

Reply:
Also, if adding `select!` we should probably also add `select`

Reply:
@nicoleepp this looks like a bug

Reply:
If that's true then please also add a test case?

Reply:
Working on fixing this